frontend/packages/helm-plugin/src/components/details-page/history/HelmReleaseHistoryHeader.tsx

```diff
@@ -1,15 +1,16 @@
+import * as classNames from 'classnames';
 import { TFunction } from 'i18next';
 import { sortable } from '@patternfly/react-table';
 import { Kebab } from '@console/internal/components/utils';
 
 export const tableColumnClasses = {
-  revision: 'col-lg-1 col-md-3 col-sm-3 col-xs-3',
-  updated: 'col-lg-2 col-md-3 col-sm-5 col-xs-5',
-  status: 'col-lg-1 col-md-2 hidden-sm hidden-xs',
-  chartName: 'col-lg-2 hidden-md hidden-sm hidden-xs',
-  chartVersion: 'col-lg-2 hidden-md hidden-sm hidden-xs',
-  appVersion: 'col-lg-2 hidden-md hidden-sm hidden-xs',
-  description: 'col-lg-2 hidden-md hidden-sm hidden-xs',
+  revision: classNames('pf-u-w-8-on-lg', 'pf-u-w-25-on-xs'),
```

Comment:
You can continue to use a space separated string instead of the `classnames` module. It adds no value here.

Reply:
noted